src/ducks/client/mobile/mobile.spec.js

```diff
@@ -0,0 +1,10 @@
+import { getClient } from './mobile'
+
+describe('get mobile client', () => {
+  it('should correctly register plugins', () => {
```

Comment:
it should correctly have excepted registered plugins ?

Reply:
I don't understand your question

Reply:
Done via private message, the name of test could be closer to the fact that we do not test that plugin are registered (testing implementation), but we test that they are there at the end. Fixed to "should have plugins correctly instantiated" in another commit.